git-cliff-core/src/changelog.rs

```diff
@@ -54,12 +55,19 @@ impl<'a> Changelog<'a> {
 				None => None,
 			},
 			config,
+			additional_context: HashMap::new(),
 		};
 		changelog.process_commits();
 		changelog.process_releases();
 		Ok(changelog)
 	}
 
+	/// Add a key value pair that will be added to the tera context
+	/// when generating the changelog.
+	pub fn add_context(&mut self, key: impl Into<String>, value: impl Into<String>) {
```

Comment:
not sure if `String` is the correct type. Maybe you prefer `Serialize` or similar?

Reply:
I think it's fine for now!